src/calculator.m

```diff
@@ -0,0 +1,22 @@
+classdef Calculator
+    methods(Static)
```

Comment:
Consider implementing an interface or abstract class for better extensibility and adherence to design principles.